core/snippets/snippets_insert_raw_text.py

```diff
@@ -0,0 +1,50 @@
+import re
+
+from talon import actions
+
+
+def insert_snippet_raw_text(body: str):
+    """Insert snippet as raw text without editor support"""
+    lines = body.splitlines()
+    found_stop = False
+
+    for i, line in enumerate(lines):
+        # Some IM services will send the message on a tab
+        line = re.sub(r"\t+", "    ", line)
+
+        if found_stop != "$1":
+            if "$1" in line:
+                found_stop = "$1"
+                stop_row = i
+                stop_col = line.index("$1")
+            elif "$0" in line:
+                found_stop = "$0"
+                stop_row = i
+                stop_col = line.index("$0")
+
+        # Replace placeholders with default text
+        line = re.sub(r"\$\{\d+:(.*?)\}", r"\1", line)
```

Comment:
Would be cool to support `CLIPBOARD` here 😊. If not for this PR, tho, might be at least worth a FIXME

Reply:
Agreed